developing-without-a-build/es-dev-server.md

```diff
@@ -0,0 +1,195 @@
+# Developing Without a Build: es-dev-server
+This article is part of a series on developing without a build:
+- Introduction
+- es-dev-server (this article)
+- Testing (coming soon!)
+
+In the introduction, we looked into different workflows and browser APIs. In this article, we will look into how we can set up `es-dev-server`, and how it helps with development without a build.
+
+## es-dev-server
+
+[es-dev-server](https://www.npmjs.com/package/es-dev-server) is a composable web server that focuses on developer productivity when developing with a build. Through options, you can opt into features such as caching, reloading on file changes, SPA routing, resolving bare module imports and compatibility modes to support older browsers.
```

Comment:
```suggestion
[es-dev-server](https://www.npmjs.com/package/es-dev-server) is a composable web server that focuses on developer productivity when developing with a build step. Through options, you can opt into features such as caching, reloading on file changes, SPA routing, resolving bare module imports and compatibility modes to support older browsers.
```

Reply:
Do you think we should change the title too?

Reply:
I think in the title its fine, but I think in a sentence it's a bit easier to read